products/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Joe's Stereo Shop</title>
+    <link rel="stylesheet" href="../common/main.css">
+    <link rel="stylesheet" href="products.css">
+</head>
+
+<body>
+    <header>
+        <h1>Joe's Stereo Shop</h1>
+        <nav>
+            <a href="../">Home</a>
+            <a href="products">Buy Speakers</a>
+        </nav>
+    </header>
+
+    <main>
+        <h2>Speakers</h2>
+        <ul id="speakers">
+            <li class="Wilson Audio" title="Floorstanding Loudspeaker">
+                <h3>Magnum Opus</h3><img src="../assets/wilson-audio-magnum-opus.png" alt="Wilson Speaker">
+                <p class="price">850,000.00<button>Add</button></p>
+            </li>
```

Comment:
Don't forget to remove the hard coded `<li>`. You are rendering two Wilson Audio speakers.